src/components/Settings/MenuButton/index.tsx

```diff
@@ -55,10 +57,12 @@ const ButtonContent = () => {
     )
   }
 
-  const isInvalidSlippage = validateUserSlippageTolerance(userSlippageTolerance) !== SlippageValidationResult.Valid
+  const tooHigh = userSlippageTolerance.greaterThan(MAXIMUM_RECOMMENDED_SLIPPAGE)
```

Comment:
If I'm reading this correctly, it looks like you're removing the only current use of `validateUserSlippageTolerance`. Are you sure we want to remove the slippage too low case here?

```
  if (userSlippageTolerance.lessThan(MINIMUM_RECOMMENDED_SLIPPAGE)) {
    return SlippageValidationResult.TooLow
  }
```

Reply:
If this is intended and it is the last usage of the function, then we should remove the file `src\utils\validateUserSlippageTolerance.ts`